modules/update-ensuring-the-host-firmware-is-compatible.adoc

```diff
@@ -1,22 +1,22 @@
 // Module included in the following assemblies:
 //
-// * edge_computing/day_2_core_cnf_clusters/updating/telco-update-ocp-update-prep.adoc
+// * edge_computing/day_2_core_cnf_clusters/updating/update-ocp-update-prep.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="telco-update-ensuring-host-firmware-compatible_{context}"]
+[id="update-ensuring-host-firmware-compatible_{context}"]
 = Ensuring the host firmware is compatible with the update
 
 You are responsible for the firmware versions that you run in your clusters.
 Updating host firmware is not a part of the {product-title} update process.
-It is not recommended to update firmware in conjunction
-with the {product-title} version.
+It is not recommended to update firmware in conjunction with the {product-title} version.
 
 [IMPORTANT]
 ====
 Hardware vendors advise that it is best to apply the latest certified firmware version for the specific hardware that you are running.
-For telco use cases, always verify firmware updates in test environments before applying them in production.
-The high throughput nature of telco CNF workloads can be adversely affected by sub-optimal host firmware.
+For each different use case, always verify firmware updates in test environments before applying them in production.
+For example: the high throughput nature of some workloads can be adversely affected by sub-optimal host firmware.
```

Comment:
Consider re-writing, perhaps something like the following:

For example, workloads with high throughput requirements might feature reduced performance because of outdated host firmware.